tools/deb_to_rb.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+set -e
+
+CREW_PREFIX=${CREW_PREFIX:-/usr/local}
+CREW_LIB_PREFIX=${CREW_PREFIX}/lib
+PKG_PREFIX=${CREW_LIB_PREFIX}/crew/packages
+GREEN='\e[1;32m';
+RED='\e[1;31m';
+BLUE='\e[1;34m';
+YELLOW='\e[1;33m';
+RESET='\e[0m'
+
+rm -rf ${CREW_PREFIX}/tmp/deb && cd ${CREW_PREFIX}/tmp/
+
+mkdir -p deb && cd ./deb
+
+echo -e "${YELLOW}Downloading deb...${RESET}"
+curl '-#' ${1} -o deb
+
+ar x deb
+
+
+tar xf control.tar.*
```

Comment:
Is every extracted deb file named `control.tar.*`?

Reply:
Yeah, that file may be 'control.tar.xz' or 'control.tar.gz'